iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceMethodCommon.java

```diff
@@ -59,8 +58,6 @@ public static Collection inputs() throws Exception
 
     protected static String iotHubConnectionString = "";
 
-    protected static RegistryManager registryManager;
```

Comment:
Making each test have their own registry manager instead of sharing one static instance